lib/Conversion/XTenNNToTorch.cpp

```diff
@@ -37,6 +37,19 @@ using namespace mlir::torch;
 
 namespace {
 
+Type toTorchTensorTypeCast(PatternRewriter &rewriter, Type ty) {
```

Comment:
So the only thing that was not handled properly is signless integers? So torch mlir does always expected either singed or unsigned integers?

Reply:
Discussed offline